src/components/tl/mlx5/tl_mlx5_team.c

```diff
@@ -195,7 +185,7 @@ ucc_status_t ucc_tl_mlx5_team_get_scores(ucc_base_team_t *  tl_team,
         MAX_MSG_SIZE * UCC_TL_TEAM_SIZE(team), UCC_TL_MLX5_DEFAULT_SCORE,
         ucc_tl_mlx5_coll_init, tl_team);
     if (UCC_OK != status) {
-        tl_error(lib, "failed to add range to score_t");
+        tl_debug(lib, "failed to add range to score_t");
```

Comment:
why do you manually add score ranges to table instead of calling ucc_coll_score_build_default?

Reply:
fixed as you suggest